waku/common/databases/db_postgres/dbconn.nim

```diff
@@ -75,32 +76,82 @@ proc sendQuery(db: DbConn,
 
   return ok()
 
+proc sendQueryPrepared(
+               db: DbConn,
+               stmtName: string,
+               paramValues: openArray[string],
+               paramLengths: openArray[int32],
+               paramFormats: openArray[int32]):
+               Result[void, string] =
+  ## This proc can be used directly for queries that don't retrieve values back.
+
+  if paramValues.len != paramLengths.len or paramValues.len != paramFormats.len or
+     paramLengths.len != paramFormats.len:
+    let lengthsErrMsg = $paramValues.len & " " & $paramLengths.len & " " & $paramFormats.len
+    return err("lengths discrepancies in sendQueryPrepared: " & $lengthsErrMsg)
```

Comment:
Good catch! Thanks for it!

The thing is that we need to keep those three arrays separately so that we can pass their addresses to https://www.postgresql.org/docs/current/libpq-async.html#LIBPQ-PQSENDQUERYPREPARED

If you agree, we could clean that up by defining a `ParamList` type and then store the lists there.

Reply:
> The thing is that we need to keep those three arrays separately so that we can pass their addresses to https://www.postgresql.org/docs/current/libpq-async.html#LIBPQ-PQSENDQUERYPREPARED

:thinking: 

> If you agree, we could clean that up by defining a `ParamList` type and then store the lists there.

That would kinda defeat the purpose.

Disregard my comment, it's not really practical.